data-es6.js

```diff
@@ -66,12 +66,10 @@ exports.browsers = {
     short: 'IE 11',
     obsolete: false
   },
-  ie11tp: {
-    full: 'Internet Explorer',
-    short: 'IE<br>Technical<br>Preview',
-    unstable: true,
-    note_id: 'ie-experimental-flag',
-    note_html: 'Have to be enabled via "Experimental Web Platform Features" flag'
+  spartan: {
+    full: 'Project Spartan',
```

Comment:
Well, what's important is the JS engine, and I believe it still uses Chakra (or at the very least inherits all of the functionality of Chakra, such that an IE10/11 `true` remains so for Spartan unless otherwise noted).